src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIPipelineService.java

```diff
@@ -74,8 +92,21 @@ public UIPipelineDetailsResponse getPipelineDetails(UUID id, Locale locale) thro
          */
 		detailsResponse.setSavedPipelineParameters(getSavedPipelineParameters(workflow, locale));
 
-		return detailsResponse;
-	}
+		/*
+        Check / add reference files
+         */
+        if (description.requiresReference()) {
+            /*
+            Need to get a list of all the projects in the cart
```

Comment:
This comment needs to be fixed as well. Looks like there is a missing `*` before the comment text

Reply:
I see your issue.  Those are not incorrectly formatted comments, they are just considered "multiline" comments in Java.  They are not supoosed to be JavaDoc.

Reply:
Makes sense!